decentraland/unity-explorer
# feat: added ocean shader in satellite map rendering

## What does this PR change?

This PR adds the ocean shader and its integration in the satellite map

...

## How to test the changes?



1. Launch the explorer
2. Open the navmap
3. Check that the ocean is properly rendered

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
